core/SConscript.bootloader_ci

```diff
@@ -74,7 +74,9 @@ CPPPATH_MOD += [
         'vendor/micropython/lib/uzlib',
 ]
 SOURCE_MOD += [
-    'embed/lib/display.c',
+    'embed/lib/display_draw.c',
+    'embed/lib/display_utils.c',
+    'embed/lib/terminal.c',
```

Comment:
lets sort this like the other Sconscripts

Reply:
I've sorted this section in all sconscripts, see 9cb7cb23918e51137acd82b42aa1f5e39db36b2d